src/clj/puppetlabs/services/ca/certificate_authority_core.clj

```diff
@@ -14,6 +14,7 @@
             [liberator.core :refer [defresource]]
             ;[liberator.dev :as liberator-dev]
```

Comment:
Yeah, I actually uncommented that to debug this one initially, too, so guess it's still serving its purpose?  :smile: